pkgs/tools/misc/mat2/default.nix

```
@@ -0,0 +1,40 @@
+{ python3Packages, fetchurl, lib, pkgs }:
```

```suggestion
{ python3Packages, fetchurl, lib, ... }:
```

pkgs shouldn't be used as an input and should be replaced with the actual packages.